Add installation rpath - this is robotpkg specific (ros has no install).

--- rxtools/CMakeLists.txt~	2010-09-09 00:00:26.000000000 +0200
+++ rxtools/CMakeLists.txt	2012-02-20 16:09:53.000000000 +0100
@@ -80,6 +80,14 @@
 rosbuild_add_executable(rxconsole src/rxconsole/rxconsole.cpp)
 target_link_libraries(rxconsole ${PROJECT_NAME})
 
+get_target_property(_rpaths rxconsole INSTALL_RPATH)
+list(APPEND _rpaths "$ENV{PREFIX}/$ENV{ROS_INSTALLATION_SUBDIR}/rxtools/lib")
+set_target_properties(rxconsole PROPERTIES INSTALL_RPATH "${_rpaths}")
+
 #rxloggerlevel binary
 rosbuild_add_executable(rxloggerlevel src/rxloggerlevel/rxloggerlevel.cpp)
 target_link_libraries(rxloggerlevel ${PROJECT_NAME})
+
+get_target_property(_rpaths rxloggerlevel INSTALL_RPATH)
+list(APPEND _rpaths "$ENV{PREFIX}/$ENV{ROS_INSTALLATION_SUBDIR}/rxtools/lib")
+set_target_properties(rxloggerlevel PROPERTIES INSTALL_RPATH "${_rpaths}")
